src/components/tabs/tabs.stories.tsx

```diff
@@ -0,0 +1,1660 @@
+import React, { useState } from "react";
+import { Tabs, Tab } from ".";
+import { Checkbox } from "../checkbox";
+import Pill from "../pill";
+import Icon from "../icon";
+import Box from "../box";
+
+export const Default = () => {
+  return (
+    <div style={{ padding: "4px" }}>
```

Comment:
Considering existence of this PR - https://github.com/Sage/carbon/pull/5869 - maybe it's worth replacing the divs with Boxes?

Reply:
Oh yeah I see what you mean, this won't get merged in due to it being a different file. Thanks, I was unaware of this - might as well do it when the change is so small.